core-lib/Actors.ns

```diff
@@ -258,9 +267,6 @@ class Actors usingVmMirror: vmMirror usingKernel: kernel = Value (
   )
 
   public after: milliseconds do: aBlock = (
-    | promisePair |
-    promisePair:: createPromisePair.
-    promisePair promise whenResolved: aBlock.
-    vmMirror actorResolveProm: (promisePair resolver) after: milliseconds.
+      timer <-: after: milliseconds do: aBlock.
```

Comment:
Why is this an async send? This is going to register a callback, why can't this be synchronous?
Again, why do you need a timer object here?